PassiFyTests/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="NUnit" version="3.11.0" targetFramework="net452" />
```

Comment:
I installed NUnit Nuget Package, so you will need to install that/it will install when you build.